addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/tankerkoenig.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tankerkoenig"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+        <!-- Sample Thing Type -->
+        <thing-type id="tankstelle">
```

Comment:
Well, we are talking about an "initial contribution", so there are no official users of that binding so far.
But yes, anybody using the old code would have to update their thing file accordingly (or recreate things in the Paper UI).